src/ast/nodes/UnaryExpression.ts

```diff
@@ -69,4 +83,56 @@ export default class UnaryExpression extends NodeBase {
 			this.scope.context.requestTreeshakingPass();
 		}
 	}
+
+	include(
+		context: InclusionContext,
+		includeChildrenRecursively: IncludeChildren,
+		_options?: InclusionOptions
+	): void {
+		if (!this.deoptimized) this.applyDeoptimizations();
+		this.included = true;
+		const literalValue = includeChildrenRecursively
+			? UnknownValue
+			: this.getLiteralValueAtPath(EMPTY_PATH, SHARED_RECURSION_TRACKER, this);
+		if (typeof literalValue === 'symbol' || this.argument.shouldBeIncluded(context)) {
+			this.argument.include(context, includeChildrenRecursively);
+			this.literalValue = UnknownValue;
+		} else {
+			this.literalValue = literalValue;
+		}
```

Comment:
To slightly improve performance, we could run this logic only if `this.literalValue` does not have a value yet. Maybe extract it to a helper method, e.g. `this.getLiteralValue()`?

Reply:
Okay, I get it!